src/controllers/orbit-controller/orbit-state.js

```diff
@@ -0,0 +1,377 @@
+import OrbitViewport from '../../lib/viewports/orbit-viewport';
+import vec3_add from 'gl-vec3/add';
```

Comment:
> The Vector3 class requires luma.gl as a dependency.

No worries, we will break out the math package as well. Most likely luma.gl will become a monorepo, publishing multiple modules, but we could make it into a separate repo.